tests/UnitBLAS/Level2.hs

```diff
@@ -7,7 +7,6 @@
 
 module UnitBLAS.Level2(unitTestLevel2BLAS) where 
 
-import Test.HUnit
 --import Numerical.Array.Shape as S 
```

Comment:
why did you remove the Hunit import?
(meta aside: at some point soon i think i'll be migrating from tasty to hspec, perhaps )